### ISSUE TITLE
Some warnings in google colab 

### ISSUE THREAD

**Issue by wilzamguerrero** [ddPn08/kohya-sd-scripts-webui#6]
Hi thanks for this repo and all the work behind, i appreciate too much work at the end but i have this warnings i odnt know if this affect for teh proccess

################################################################
Create and activate python venv
################################################################

################################################################
Launching launch.py...
################################################################
Installing torch and torchvision
Installing xformers
Installing gradio
Installing ngrok
Installing requirements for kohya sd-scripts
2023-02-13 17:03:25.840759: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-02-13 17:03:26.109682: E tensorflow/stream_executor/cuda/cuda_blas.cc:2981] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2023-02-13 17:03:27.283729: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /content/kohya-sd-scripts-webui/venv/lib/python3.10/site-packages/cv2/../../lib64:/usr/lib64-nvidia
2023-02-13 17:03:27.283989: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /content/kohya-sd-scripts-webui/venv/lib/python3.10/site-packages/cv2/../../lib64:/usr/lib64-nvidia
2023-02-13 17:03:27.284022: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
A matching Triton is not available, some optimizations will not be enabled.
Error caught was: No module named 'triton'

To create a public link, set `share=True` in `launch()`.
Started Python: /content/kohya-sd-scripts-webui/kohya_ss/finetune/make_captions.py
Arguments:  ['--pretrained_model_name_or_path', '/content/sample_data/Zv1-5-fp16.ckpt', '--train_data_dir', '/content/sample_data/arargue2', '--shuffle_caption', '--caption_extension', '.caption', '--flip_aug', '--resolution', '512', '--enable_bucket', '--min_bucket_reso', '256', '--max_bucket_reso', '1024', '--dataset_repeats', '1', '--output_dir', '/content/sample_data', '--output_name', 'arargue', '--save_precision', 'fp16', '--save_every_n_epochs', '5', '--train_batch_size', '1', '--use_8bit_adam', '--xformers', '--learning_rate', '0.0001', '--max_train_steps', '1600', '--max_data_loader_n_workers', '8', '--gradient_checkpointing', '--gradient_accumulation_steps', '1', '--mixed_precision', 'no', '--clip_skip', '2', '--lr_scheduler', 'constant', '--prior_loss_weight', '1.0', '--save_model_as', 'safetensors', '--network_module', 'networks.lora', '--network_dim', '16', '--network_alpha', '1.0']

**Comment by ddPn08** [comment]
You can safely ignore these warnings.
